port_flatpak.patch

```diff
@@ -0,0 +1,166 @@
+     def draw_score(self, cr):
+         cr.set_source_rgb(0, 0, 0)
+         pango_layout = PangoCairo.create_layout(cr)
+-        fd = Pango.FontDescription('Times')
++        fd = Pango.FontDescription('Monospace')
+         fd.set_size(14 * Pango.SCALE)
+         pango_layout.set_font_description(fd)
+         text = _('SCORE: %d') % self.score
+@@ -380,7 +380,7 @@ class BalloonGame(Gtk.VBox):
+         # Draw instructions.
+         cr.set_source_rgb(0, 0, 0)
+         pango_layout = PangoCairo.create_layout(cr)
+-        pango_layout.set_font_description(Pango.FontDescription('Times 14'))
++        pango_layout.set_font_description(Pango.FontDescription('Monospace 14'))
+         text = _('Type the words to pop the balloons!')
+         pango_layout.set_text(text, len(text))
+         size = pango_layout.get_size()
+diff --git a/editlessonscreen.py b/editlessonscreen.py
+index 8d7e8ec..5c73d5a 100644
+--- a/editlessonscreen.py
++++ b/editlessonscreen.py
+@@ -543,7 +543,7 @@ class EditLessonScreen(Gtk.VBox):
+         # Load the myspell dictionary.
+         # TODO: Find a better way to determine its location.
+         code = locale.getdefaultlocale()[0] or 'en_US'
+-        return lessonbuilder.load_wordlist('/usr/share/myspell/%s.dic' % code)
++        return lessonbuilder.load_wordlist('/app/share/myspell/%s.dic' % code)
+ 
+     def generate_ok_clicked_cb(self, btn, box):
+         self.save()
+diff --git a/keyboard.py b/keyboard.py
+index 48bdbfc..82eb7ee 100644
+--- a/keyboard.py
++++ b/keyboard.py
+@@ -106,14 +106,7 @@ def _is_olpcm_model():
+     keyboard, and 'olpc' for membrane keyboard.
+ 
+     """
+-    code = None
+-    p = subprocess.Popen(["setxkbmap", "-query"], stdout=subprocess.PIPE)
+-    out, err = p.communicate()
+-    
+-    for line in out.decode('utf-8').splitlines():
+-        if line.startswith('model:'):
+-            code = line.split()[1]
+-    return code == 'olpcm'
++    pass
+ 
+ def get_layout():
+     if _is_olpcm_model():
+diff --git a/titlescene.py b/titlescene.py
+index b28d8b3..04b78e5 100644
+--- a/titlescene.py
++++ b/titlescene.py
+@@ -34,7 +34,7 @@ class TitleScene(Gtk.DrawingArea):
+     TITLE_OFFSET = (20, 50)
+ 
+     # Font used to display the title.
+-    TITLE_FONT = 'Times 45'
++    TITLE_FONT = 'Monospace 45'
+ 
+     def __init__(self):
+         Gtk.DrawingArea.__init__(self)
+diff --git a/typingturtle.py b/typingturtle.py
+index d37300a..df0702e 100755
+--- a/typingturtle.py
++++ b/typingturtle.py
+@@ -16,7 +16,7 @@
+ """Typing Turtle - Interactive typing tutor for Sugar."""
+ 
+ # Import standard Python modules.
+-import logging, os, math, time, copy, locale, datetime, random, re
++import logging, os, math, time, copy, locale, datetime, random, re, sys
+ from gettext import gettext as _
+ import json
+ 
+@@ -33,7 +33,7 @@ gi.require_version('Gtk', '3.0')
+ from gi.repository import Gtk
+ 
+ # Import Sugar UI modules.
+-import sugar3.activity.activity
++import sugar3
```

Comment:
is this necessary?

Reply:
Yes, it is, try removing `import sugar3` line